src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -192,7 +192,7 @@ protected bool BuildInternal (string projectOrSolution, string target, string []
 			psi.StandardErrorEncoding = Encoding.UTF8;
 			psi.StandardOutputEncoding = Encoding.UTF8;
 			var p = Process.Start (psi);
-			var ranToCompletion = p.WaitForExit ((int)new TimeSpan (0,10,0).TotalMilliseconds);
+			var ranToCompletion = p.WaitForExit ((int)new TimeSpan (0,15,0).TotalMilliseconds);
```

Comment:
Is the new mono slower and that's why we need to increase timeouts?

Reply:
This is a debugging aid I added to investigate failures I saw before the msbuild tests got broken in master. I'll take a closer look once those are fixed again.